CLA.md

```diff
@@ -18,22 +18,34 @@ This policy does not apply to [third_party] and [vendor].
 
 Click
   * [Individual signup] to sign up as an individual or as an employee of a signed organization. 
-  * [Corp signup] to sign as a corporation representative and manage signups from your organization.
+  * [Corporation signup] to sign as a corporation representative and manage signups from your organization.
 
 Either signup form looks like this:
 
 ![CNCFCLA1](http://i.imgur.com/tEk2x3j.png)
 
 #### 3. Enter the correct E-mail address to validate!
 
+Your github email address __must match__ the same address you use when signing the CLA.
+
 The address entered in the form must meet two constraints:
  
  * It __must match__ your  [git email] (the output of `git config user.email`)
    or your PRs will not be approved!
 
+ * Use `git config user.name "Mona Lisa"` and `git config user.email "mona@example.com"`
+   to ensure you've set this properly.
+
  * It must be your official `person@organization.com` address if you signed up
    as an employee of said organization.
 
+ * If you've already submitted a PR you can correct your user.name and user.email 
```

Comment:
If you've already submitted a PR you can correct your user.name and user.email
locally and then use `git commit --ammend --reset-author` and then `git push` to
correct the PR.